src/app.js

```diff
@@ -1,21 +1,17 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const { readParam } = require('./paramsHelper');
 
-module.exports = sum;
+const server = http.createServer((req, res) => {
+  if (req.url !== '/favicon.ico') {
```

Comment:
It is not a good practice to compare the url directly with '/favicon.ico'. Instead use path module to parse the url and then perform the check on the pathname.